src/hessian.F90

```diff
@@ -228,6 +228,60 @@ subroutine numhess( &
       !$ call mkl_set_num_threads(nproc)
 #endif
 
+   else if (mode_extrun.eq.p_ext_gfnff) then
```

Comment:
There is a frozen hessian case as well, see above.

Reply:
Also a `select type` is certainly better for this purpose.

Reply:
Now using `select type` and frozen hessian is added.